$OpenBSD: patch-src_tracker-preferences_tracker-preferences_ui,v 1.2 2010/07/01 17:25:21 ajacoutot Exp $

Don't display battery options as this requires HAL or DeviceKIT.

--- src/tracker-preferences/tracker-preferences.ui.orig	Thu Jul  1 17:05:57 2010
+++ src/tracker-preferences/tracker-preferences.ui	Thu Jul  1 19:04:17 2010
@@ -281,45 +281,6 @@
                               </packing>
                             </child>
                             <child>
-                              <object class="GtkCheckButton" id="checkbutton_enable_index_on_battery">
-                                <property name="label" translatable="yes">Enable when running on _battery</property>
-                                <property name="visible">True</property>
-                                <property name="can_focus">True</property>
-                                <property name="receives_default">False</property>
-                                <property name="use_underline">True</property>
-                                <property name="active">True</property>
-                                <property name="draw_indicator">True</property>
-                                <signal name="toggled" handler="checkbutton_enable_index_on_battery_toggled_cb"/>
-                              </object>
-                              <packing>
-                                <property name="expand">False</property>
-                                <property name="fill">False</property>
-                                <property name="position">1</property>
-                              </packing>
-                            </child>
-                            <child>
-                              <object class="GtkAlignment" id="alignment4">
-                                <property name="visible">True</property>
-                                <property name="left_padding">12</property>
-                                <child>
-                                  <object class="GtkCheckButton" id="checkbutton_enable_index_on_battery_first_time">
-                                    <property name="label" translatable="yes">Enable for _first time</property>
-                                    <property name="visible">True</property>
-                                    <property name="can_focus">True</property>
-                                    <property name="receives_default">False</property>
-                                    <property name="use_underline">True</property>
-                                    <property name="active">True</property>
-                                    <property name="draw_indicator">True</property>
-                                    <signal name="toggled" handler="checkbutton_enable_index_on_battery_first_time_toggled_cb"/>
-                                  </object>
-                                </child>
-                              </object>
-                              <packing>
-                                <property name="expand">False</property>
-                                <property name="position">2</property>
-                              </packing>
-                            </child>
-                            <child>
                               <object class="GtkCheckButton" id="checkbutton_index_removable_media">
                                 <property name="label" translatable="yes">Include _removable media</property>
                                 <property name="visible">True</property>
